federation/cluster/federation-up.sh

```diff
@@ -84,6 +82,6 @@ if [[ "${USE_KUBEFED}" == "true" ]]; then
   init
   create_cluster_secrets
 else
-  export FEDERATION_IMAGE_TAG="$(get_version)"
+  export FEDERATION_IMAGE_TAG="$(echo ${KUBERNETES_RELEASE:-} | tr + _)"
```

Comment:
so we had to export KUBERNETES_RELEASE to bring up federation outside CI environment?